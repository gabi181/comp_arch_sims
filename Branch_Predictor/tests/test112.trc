16 8 1 3 global_history global_tables not_using_share
0x181E6A64 N 0x504796F4
0x20162430 N 0x4AAF64EC
0x20162430 N 0x4AAF64EC
0x20162430 T 0x4AAF64EC
0x20162430 N 0x4AAF64EC
0x20162430 N 0x4AAF64EC
0x20162430 T 0x4AAF64EC
0x20162430 T 0x4AAF64EC
0x181E6A64 T 0x504796F4
