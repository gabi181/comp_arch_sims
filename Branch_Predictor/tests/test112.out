0x181e6a64 N 0x181e6a68
0x20162430 N 0x20162434
0x20162430 N 0x20162434
0x20162430 N 0x20162434
0x20162430 T 0x4aaf64ec
0x20162430 T 0x4aaf64ec
0x20162430 T 0x4aaf64ec
0x20162430 T 0x4aaf64ec
0x181e6a64 T 0x504796f4
flush_num: 3, br_num: 9, size: 1032b
